presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryCompletedEvent.java

```diff
@@ -25,7 +28,7 @@
     private final QueryContext context;
     private final QueryIOMetadata ioMetadata;
     private final Optional<QueryFailureInfo> failureInfo;
-
+    private final List<PrestoWarning> queryWarnings;
```

Comment:
Just `warnings`? It's clear that this is a query event, and so these are query warnings.